src/Compilers/Core/Portable/Operations/Loops/IForEachLoopStatement.cs

```diff
@@ -14,13 +14,20 @@ namespace Microsoft.CodeAnalysis.Semantics
     public interface IForEachLoopStatement : ILoopStatement
     {
         /// <summary>
-        /// Iteration variable of the loop.
+        /// Refers to the operation for declaring a new local variable or reference an existing variable or an expression.
         /// </summary>
-        ILocalSymbol IterationVariable { get; }
+        IOperation LoopControlVariable { get; }
```

Comment:
>LoopControlVariable [](start = 19, length = 19)

Not sure if the name works well for deconstruction scenario

Reply:
It matches the language spec though, do you have a better suggestion?